sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -210,6 +211,55 @@ public Breadcrumb(final @NotNull Date timestamp) {
     return breadcrumb;
   }
 
+  /**
+   * Creates user breadcrumb - a user interaction with your app's UI. The breadcrumb can contain
+   * additional data like {@code viewId} or {@code viewClass}. By default, the breadcrumb is
+   * captured with {@link SentryLevel.INFO} level.
+   *
+   * @param category - the category, for example "click"
+   * @param viewId - the human-readable view id, for example "button_load"
+   * @param viewClass - the fully qualified class name, for example "android.widget.Button"
+   * @return the breadcrumb
+   */
+  public static @NotNull Breadcrumb userInteraction(
+      final @NotNull String category,
+      final @Nullable String viewId,
+      final @Nullable String viewClass) {
+    return userInteraction(category, viewId, viewClass, Collections.emptyMap());
+  }
+
+  /**
+   * Creates user breadcrumb - a user interaction with your app's UI. The breadcrumb can contain
+   * additional data like {@code viewId} or {@code viewClass}. By default, the breadcrumb is
+   * captured with {@link SentryLevel.INFO} level.
+   *
+   * @param category - the category, for example "click"
+   * @param viewId - the human-readable view id, for example "button_load"
+   * @param viewClass - the fully qualified class name, for example "android.widget.Button"
+   * @param additionalData - additional properties to be put into the data bag
+   * @return the breadcrumb
+   */
+  public static @NotNull Breadcrumb userInteraction(
```

Comment:
I actually just realized this is in the core-java package - not sure if it's suitable here with all these viewId/viewClass params? We have other static methods if you look above (`user` and `ui`), but none of them really suited my case, because we don't have a message. Should I change one of the above rather?

Reply:
I think viewId and viewClass would be generic for any App that would have user interaction, so its fine.
Maybe we rename the `category` to `subCategory` since we append to the `ui.` prefix?